src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/RazorSemanticTokensEndpoint.cs

```diff
@@ -57,7 +57,9 @@ public async Task<SemanticTokens> Handle(SemanticTokensRangeParams request, Canc
             }
 
             var semanticTokens = await HandleAsync(request.TextDocument, cancellationToken, request.Range);
-            _logger.LogInformation($"Returned {semanticTokens.Data.Length / 5} semantic tokens for range {request.Range} in {request.TextDocument.Uri}.");
+            var amount = semanticTokens is null ? "no" : (semanticTokens.Data.Length / 5).ToString(Thread.CurrentThread.CurrentCulture);
+
+            _logger.LogInformation($"Returned {amount} semantic tokens for range {request.Range} in {request.TextDocument.Uri}.");
```

Comment:
If the result is null my silly log will throw because we haven't turned on nullable yet.